kubernetes-jenkins-plugin/kube-fcos-pipeline.yaml

```diff
@@ -4,6 +4,26 @@ apiVersion: "build.openshift.io/v1"
 metadata:
   name: "kubernetes-fcos-pipeline"
 spec:
+  # Define triggers.
+  #   - trigger on buildconfig change
+  #   - trigger on imagestream change
+  #   - trigger on GitHub webhook
+  #   - trigger on Generic webhook (for fedmsg triggered actions)
+  triggers:
+  - type: ConfigChange
+  # use `oc set triggers --from-github` to update token to be unique
+  - type: GitHub
+    github:
+      secret: regenerate
```

Comment:
You can also switch this file to be a template and then define this value to be a parameter which defaults to e.g.:

```
- description: Shared webhook secret.
  name: GITHUB_WEBHOOK_SECRET
  generate: expression
  from: '[A-Z0-9]{32}'
  required: true
```

The advantage of this is that folks can set their webhook secret at `oc create` time so that they don't have to keep changing webhook secrets in their fork settings when testing. E.g. `oc process -p GITHUB_WEBHOOK_SECRET=... | oc create -f -`.

Reply:
yeah I thought about that. For some reason I've tried to shy away from openshift templates in the past but I guess since we're using a buildconfig here (openshift specific) it wouldn't hurt to make it an openshift template.

I'll put that as a future thing to do since I don't think I'm going to get around to it today. Will handle in a future PR.